App/Sagas/TextileSagas.js

```diff
@@ -128,8 +123,9 @@ export function * triggerStopNode () {
 
 export function * createNode ({path}) {
   try {
-    const debugLevel = (__DEV__ ? 'DEBUG' : 'INFO')
-    const createNodeSuccess = yield call(IPFS.createNodeWithDataDir, path, API_URL, debugLevel)
+    const logLevel = (__DEV__ ? 'DEBUG' : 'INFO')
+    const logFiles = !__DEV__
```

Comment:
Does `logFiles === true` mean that the log output shows up in Xcode or not? Kind of a strange name.

Reply:
covered in our call, but adding here for future ref... `logFiles` is a boolean indicating whether or not to write log output to actual files on the file system.

Reply:
(if false, they will instead be written to stdout)